src/modules/auth/ui/views/sign-in-view.tsx

```
@@ -0,0 +1,230 @@
+                      <div className="absolute inset-0 flex items-center justify-center">
+                        <div className="w-6 h-6 border-2 border-white border-t-transparent rounded-full animate-spin mr-2"></div>
+                        Signing In...
+                      </div>
+                    )}
+                    <div className="absolute inset-0 bg-gradient-to-r from-pink-600 via-purple-600 to-orange-500 opacity-0 group-hover:opacity-100 transition-opacity duration-300"></div>
+                  </Button>
+
+                  {/* Sign Up Link */}
+                  <div className="text-center mt-6 pt-6 border-t border-gray-200">
+                    <p className="text-gray-600">
+                      Don't have an account?{' '}
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)